onsite-group-location.html

```diff
@@ -5,116 +5,77 @@
-      <p class="flush-bottom push-quarter-top small"><strong>Wednesday 6:30-8:30 PM</strong></p>
-      <button class="btn btn-cyan push-top" type="button">Register</button>
-    </div>
-  </div>
-
-  <div class="row push-ends soft-bottom onsite-group-negative-margin-bottom">
-    <div class="col-md-3">
-      <h4 class="font-family-condensed-extra text-uppercase">Senior Connection</h4>
-      <p class="text-gray flush-bottom small"><strong>Perfect for:</strong> Anyone looking to study God's Word</p>
-      <p class="flush-bottom push-quarter-top small"><strong>Day varies 6:30-8:30 PM</strong></p>
-      <button href="#" type="button" class="btn btn-link hard-left" data-placement="bottom" data-toggle="tooltip"
-        title="seniorconnection.mason@crossroads.net">Email
-        for more info</button>
-    </div>
+  <div class="row push-bottom onsite-group-negative-margin-bottom">
+    {% for meeting in page.meetings %}
     <div class="col-md-3">
-      <h4 class="font-family-condensed-extra text-uppercase">Ocean</h4>
-      <p class="text-gray flush-bottom small"><strong>Perfect for:</strong> Any entrepreneur, artist, business owner, or
-        any leader interested in launching something new, and looking for a community to support their vision. </p>
-      <p class="flush-bottom push-quarter-top small"><strong>Thursday 6:30-8:30 PM</strong></p>
-      <button class="btn btn-cyan push-top" type="button">Register</button>
+      <h4 class="font-family-condensed-extra text-uppercase">
+        {{ meeting.title }}
+      </h4>
+      <p class="text-gray flush-bottom small">{{ meeting.description }}</p>
+      <p class="flush-bottom push-quarter-top small">
+        {{ meeting.meeting_time }}
+      </p>
+      <a
+        href="{{ meeting.registration_link }}"
+        class="btn btn-cyan push-top"
+        type="button"
+        >Register</a
+      >
     </div>
+    {% endfor %}
   </div>
 
-  <hr>
+  <hr />
   <div class="row soft-half-bottom soft-top">
     <div class="col-md-6">
-      <p><strong>Can't find what you are looking for?</strong><br>
-        Visit the <a href="#">Group Tool</a> to connect with others looking for the same thing.</p>
+      <p>
+        <strong>Can't find what you are looking for?</strong><br />
+        Visit the <a href="#">Group Tool</a> to connect with others looking for
+        the same thing.
+      </p>
     </div>
     <div class="col-md-6">
       <p>Questions? Email us at <a href="#">hello@crossroads.net</a></p>
-      <p> If you need immediate care, please contact our team for a Listening
-        Appointment at <a href="#">hello@crossroads.net</a></p>
+      <p>
+        If you need immediate care, please contact our team for a Listening
+        Appointment at <a href="#">hello@crossroads.net</a>
+      </p>
     </div>
   </div>
 </section>
 
-<script> 
-$(document).ready(function(){
+<script>
+  $(document).ready(function() {
     $('[data-toggle="tooltip"]').tooltip();
-});
+  });
```

Comment:
Should we make this a global utility that runs on every page? Seems nice and reusable.